docs/src/content/docs/da/environmental-impact.md

```diff
@@ -0,0 +1,128 @@
+
+Caching bruges til at kontrollere, hvor længe en browser gemmer og genbruger data, den allerede har downloadet. En god caching-strategi sikrer, at en bruger får nyt indhold så hurtigt som muligt, når det ændres, men undgår også at downloade det samme indhold unødvendigt, når det ikke er ændret.
+
+Den mest almindelige måde at konfigurere caching på er med [`Cache-Control` HTTP-headeren][cache]. Når du bruger Starlight, kan du sætte en lang cachetid for alt i `/_astro/`-mappen. Denne mappe indeholder CSS, JavaScript og andre bundne assets, der kan caches sikkert for evigt, hvilket reducerer unødvendige downloads:
+
+```
+Cache-Control: public, max-age=604800, immutable
+```
+
+Hvordan man konfigurerer caching afhænger af din webhost. For eksempel anvender Vercel denne caching-strategi for dig uden behov for konfiguration, mens du kan sætte [tilpassede headers for Netlify][ntl-headers] ved at tilføje en `public/_headers`-fil til dit projekt:
+
+```
+/_astro/*
+  Cache-Control: public
+  Cache-Control: max-age=604800
+  Cache-Control: immutable
+```
+
+[cache]: https://csswizardry.com/2019/03/cache-control-for-civilians/
+[ntl-headers]: https://docs.netlify.com/routing/headers/
+
+## Strømforbrug
+
+Hvordan en webside er bygget, kan påvirke, hvor meget strøm det kræver at køre den på en brugers enhed. Ved at bruge minimalt JavaScript reducerer Starlight den mængde behandlingskraft, som en brugers telefon, tablet eller computer har brug for til at indlæse og gengive sider.
+
+Vær opmærksom på, når du tilføjer funktioner som analytik-tracking scripts eller JavaScript-tungt indhold som videoindlejringer, da disse kan øge sideens strømforbrug. Hvis du har brug for analytik, kan du overveje at vælge en letvægtsløsning som [Cabin][cabin], [Fathom][fathom] eller [Plausible][plausible]. Indlejringer som YouTube- og Vimeo-videoer kan forbedres ved at vente med at [indlæse videoen ved brugerinteraktion][lazy-video]. Pakker som [`astro-embed`][embed] kan hjælpe med almindelige tjenester.
+
+:::tip[Vidste du?]
+Parsing og kompilering af JavaScript er en af de mest ressourcekrævende opgaver, som browsere skal udføre. Sammenlignet med at gengive et JPEG-billede af samme størrelse kan [JavaScript tage mere end 30 gange længere tid at behandle][cost-of-js].
+:::
+
+[cabin]: https://withcabin.com/
+[fathom]: https://usefathom.com/
+[plausible]: https://plausible.io/
+[lazy-video]: https://web.dev/iframe-lazy-loading/
+[embed]: https://www.npmjs.com/package/astro-embed
+[cost-of-js]: https://medium.com/dev-channel/the-cost-of-javascript-84009f51e99e
+
+## Hosting
+
+Hvor en webside er hostet, kan have stor indflydelse på, hvor miljøvenlig din dokumentationsside er. Datacentre og serverfarme kan have en stor økologisk indflydelse, herunder højt elforbrug og intensiv brug af vand.
+
+At vælge en host, der bruger vedvarende energi, vil betyde lavere CO₂-udledninger for dit site. [Green Web Directory][gwb] er et værktøj, der kan hjælpe dig med at finde hostingfirmaer.
```

Comment:
Same as before regarding CO₂ in this sentence to keep consistency with the English version.